src/constraints/outage_constraints.jl

```diff
@@ -30,8 +30,8 @@
 function add_dv_UnservedLoad_constraints(m,p)
     # effective load balance (with slack in dvUnservedLoad)
     @constraint(m, [s in p.s.electric_utility.scenarios, tz in p.s.electric_utility.outage_start_time_steps, ts in p.s.electric_utility.outage_time_steps],
-        m[:dvUnservedLoad][s, tz, ts] >= p.s.electric_load.critical_loads_kw[tz+ts]
-        - sum(  m[:dvMGRatedProduction][t, s, tz, ts] * p.production_factor[t, tz+ts] * p.levelization_factor[t]
+        m[:dvUnservedLoad][s, tz, ts] >= p.s.electric_load.critical_loads_kw[tz+ts-1]
```

Comment:
Your description of the three sets is correct. The number of time steps does change from scenario to scenario, and p.s.electric_utility.outage_time_steps goes up to the longest outage duration (see [src/core/electric_utility.jl](https://github.com/NREL/REopt.jl/blob/outages/src/core/electric_utility.jl#L43)). Instead of what you're suggesting, what we currently do is handle this when summing up outage costs. Looking at [src/constraints/outage_constraints.jl](https://github.com/https://github.com/NREL/REopt.jl/blob/outages/src/constraints/outage_constraints.jl#L53), for each s in p.s.electric_utility.scenarios, only time steps up to the appropriate duration p.s.electric_utility.outage_durations[s] are included in the cost sum. So load can be unserved in those uncounted time steps with no cost. I'm not arguing that this approach is the best way, but I believe it does achieve the intended result. It would be interesting to test out if an implementation like what you're describing improves solve times.

Reply:
Ok, this makes sense, and after testing a couple different ways I agree this does what's intended.  I added new issue https://github.com/NREL/REopt.jl/issues/170 as a note to revisit this.